ui/src/app.tsx

```diff
@@ -90,7 +91,22 @@ export const App = () => (
           <Route path="collections/:id" element={<CollectionDetails />} />
           <Route path="*" element={<UnderConstruction />} />
         </Route>
-        <Route path="/:slug" element={<InfoPageContainer />} />
```

Comment:
We will no longer try to render an info page for generic slugs. This is a left over from when the backend returned dynamic content here.

Now we require exact matches!